dbms/src/Flash/Mpp/tests/gtest_mpp_exchange_writer.cpp

```diff
@@ -408,6 +410,7 @@ try
         *dag_context_ptr);
     for (const auto & block : blocks)
         dag_writer->write(block);
+    dag_writer->flush();
     dag_writer->finishWrite();
```

Comment:
maybe we can also call `flush` inside `finishWrite`, so these test code don't need to be changed.

Reply:
For example, "testBatchWriteFineGrainedShuffle" test, expect the write and finishWrite only produce one packet.

Reply:
It looks like that these tests are meaningless since we now will always flush?

Reply:
Yeah, just invoke it as a test.